### PR TITLE
added custom pipeline and solved exercise

### PR DESCRIPTION
**Pull request by Shikha2105**
solved exercise questions and created custom transfoemer.

### CODE REVIEW

File: housing.ipynb
Comment (tigersharmadivyansh2000) [inline housing.ipynb:1]: In cell 27, line "y_imp = preprocessing.fit(X).get_feature_names_out()", got the error as Error message

AttributeError: Estimator impute does not provide get_feature_names_out. Did you mean to call pipeline[:-1].get_feature_names_out()?

**Review by tigersharmadivyansh2000 — CHANGES_REQUESTED** [review]
Make changes in housing.ipynb

File: housing.ipynb
Comment (Shikha2105) [reply housing.ipynb:1]: image
I have ran that cell and it is working fine for me . Please run all cells in sequential manner .

Comment (tigersharmadivyansh2000) [reply housing.ipynb:1]: Run all, runs the notebook in sequential manner only. I was able to run after removing the some values in param grid
image

**Review by tigersharmadivyansh2000 — APPROVED** [review]
Looks good

**Pull request merged**